src/Razor/Microsoft.AspNetCore.Razor.Language/test/IntegrationTests/ComponentCodeGenerationTestBase.cs

```diff
@@ -3353,7 +3353,7 @@ public class MyComponent<TItem> : ComponentBase
             // Act
             var generated = CompileToCSharp(@"
 <MyComponent TItem=string Item=""@(""hi"")"">
-  <div>@context.ToLower()</div>
+  <div>@context.ToLowerInvariant()</div>
```

Comment:
Hm, seems like these razor tests are failing on macOS with this change:
```
Microsoft.AspNetCore.Razor.Language.IntegrationTests.IntermediateNodeVerifier+Walker+IntermediateNodeBaselineException : Locations are not equal.\n\nExpected:                 Component - (0:0,0 [90] x:\dir\subdir\Test\TestComponent.cshtml) - MyComponent\nActual:                 Component - (0:0,0 [99] x:\dir\subdir\Test\TestComponent.cshtml) - MyComponent\n\nPath:\nMicrosoft.AspNetCore.Razor.Language.Intermediate.MethodDeclarationIntermediateNode\nMicrosoft.AspNetCore.Razor.Language.Intermediate.ClassDeclarationIntermediateNode\nMicrosoft.AspNetCore.Razor.Language.Intermediate.NamespaceDeclarationIntermediateNode\nMicrosoft.AspNetCore.Razor.Language.Intermediate.DocumentIntermediateNode\n

```

@pranavkm do you think these should not be touched?

Reply:
If we do edit these files, the baselines need to be generated. That said, the code doesn't run, so it might be best to leave these tests alone.